boards/arm/nucleo_g431rb/nucleo_g431rb.dts

```diff
@@ -51,6 +51,33 @@
 	};
 };
 
+&clk_hse{
+	clock-frequency = <DT_FREQ_M(24)>;
+	status = "okay";
+};
+
+&pll {
+	/* 150MHz output */
+	/* div-m = <4>; */
+	/* mul-n = <50>; */
+	/* 170MHz output */
```

Comment:
"boost power mode" comment has been removed, was it relevant?

Reply:
I finally removed the whole comment.